pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -409,29 +409,6 @@ func (a Adapter) createOrUpdateComponent(componentExists bool, ei envinfo.EnvSpe
 		return err
 	}
 
-	// Get ServiceBinding Secret name for each Link
```

Comment:
If the cluster does not support SSA, the link information in the container will be erased, right?

Reply:
Yes, and No :-) If the cluster doesn't support SSA there will be no `ServiceBinding` becasue Service Binding Operator can't work without it.